electronjs/fillr.js

```diff
@@ -15,9 +15,17 @@ const interval = setInterval(() => {
     const profileDataHandler = new ProfileDataInterface((mappings) => {
       mappings.profile = ProfileData; // Set your profile data
       fillr.performFill(mappings);
-      console.log(fillr.getApiState().toString()) // Check api state
     })
-    
     const fillr = new FillrController.default(devKey, secretKey, profileDataHandler);
+
+    const FillrScraper = require('@fillr_letspop/cart-scraper')
+    window.FillrCartInformationExtractionInterface.setDevKey(devKey);
+    const onCartDetected = function(ev) {
+
+      const cartInfo = ev.detail;
+      alert(JSON.stringify(cartInfo));
```

Comment:
Please add the error handling for JSON

Reply:
@mhussain This is just for the example showing the log.